SU2_CFD/src/iteration_structure.cpp

```diff
@@ -51,34 +51,34 @@ CIteration::CIteration(CConfig *config) {
 
 CIteration::~CIteration(void) { }
 
-void CIteration::SetGrid_Movement(CGeometry ****geometry_container,
-          CSurfaceMovement **surface_movement,
-          CVolumetricMovement ***grid_movement,
-          CFreeFormDefBox ***FFDBox,
-          CSolver *****solver_container,
-          CConfig **config_container,
-          unsigned short val_iZone,
-          unsigned short val_iInst,
+void CIteration::SetGrid_Movement(CGeometry **geometry,
+          CSurfaceMovement *surface_movement,
+          CVolumetricMovement *grid_movement,
+          CSolver ***solver,
+          CConfig *config,
           unsigned long IntIter,
           unsigned long ExtIter)   {
 
   unsigned short iDim;
-  unsigned short Kind_Grid_Movement = config_container[val_iZone]->GetKind_GridMovement();
+  unsigned short Kind_Grid_Movement = config->GetKind_GridMovement();
```

Comment:
As you have done here, I'm a fan of shortening the container names throughout the code to be consistent, i.e., removing '_container' from config, solver, geometry, etc. These mostly appear at the highest levels of the code hierarchy. Not pressing - we could do it slowly in passes if folks agree.

Reply:
I removed the _container from all local variables. The member variables of the driver are still called container to distinguish them from the locals.